okhttp/src/main/java/feign/okhttp/OkHttpClient.java

```diff
@@ -84,7 +84,7 @@ static Request toOkHttpRequest(feign.Request input) {
       }
     }
 
-    RequestBody body = inputBody != null ? RequestBody.create(mediaType, inputBody) : null;
+    RequestBody body = inputBody != null ? RequestBody.create(inputBody, mediaType) : null;
```

Comment:
the previous signature is deprecated

Reply:
I would say to use the deprecated one, so we keep feign compatible with a wider number or releases